alert/search.go

```diff
@@ -60,6 +60,7 @@ var searchTemplate = template.Must(template.New("search").Parse(`
 	{{end}}
 	{{ if .Search }}
 		AND (
+			CAST(a.id AS TEXT) ilike :search OR
```

Comment:
For ID searching, we should probably do an exact match instead, wyt?

Reply:
Yeah I think that makes sense, iirc we were doing that on the old endpoint anyways